.travis.yml

```diff
@@ -1,4 +1,5 @@
-script:
-
-  - mvn dependency:copy-dependencies
-  - ant -f src/main/java/com/google/security/zynamics/build.xml build-binnavi-fat-jar
+language: java
```

Comment:
Please revert this change, I'd prefer if use the same build commands in the Travis CI build as in the build instructions.